validation-test/stdlib/SequenceAlgorithm.swift

```diff
@@ -0,0 +1,29 @@
+// RUN: %target-run-stdlib-swift
+// REQUIRES: executable_test
+
+import StdlibUnittest
+import StdlibCollectionUnittest
+
+let SequenceAlgorithmTests = TestSuite("SequenceAlgorithm")
+
+SequenceAlgorithmTests.test("[1,2,3,4,5,-9,6,7,8,9,0].min()") {
+  expectEqual([1,2,3,4,5,-9,6,7,8,9,0].min()!, -9)
```

Comment:
Please use `OpaqueValue` and `MinimalComparableValue` instead of integers.

Also, there are already tests for min and max in `validation-test/stdlib/SequenceType.swift.gyb`. Please add new tests there and follow the existing pattern.